planning/behavior_velocity_planner/src/node.cpp

```diff
@@ -323,6 +323,10 @@ void BehaviorVelocityPlannerNode::onTrafficSignals(
 {
   std::lock_guard<std::mutex> lock(mutex_);
 
+  // clear previous observation
+  planner_data_.traffic_light_id_map_raw_.clear();
+  auto traffic_light_id_map_last_observed_old = planner_data_.traffic_light_id_map_last_observed_;
```

Comment:
Can this value be const?
```suggestion
  const auto traffic_light_id_map_last_observed_old = planner_data_.traffic_light_id_map_last_observed_;
```

Reply:
Thank you for your comments! I reflected your comment